README.md

```diff
@@ -135,16 +135,16 @@ $ st-flash --reset --connect-under-reset write build/zephyr/zephyr.signed.confir
 
 ```bash
 $ st-flash --reset --connect-under-reset erase
-$ st-flash --reset --connect-under-reset write LexxHard-MainBoard-Firmware-Initial-v?.?.? 0x8000000
+$ st-flash --reset --connect-under-reset write LexxHard-SensorControlBoard-Firmware-Initial-v?.?.? 0x8000000
 ```
 
 ## Update via ROS
 
-Use [LexxPluss/LexxHard-MainBoard-Updator](https://github.com/LexxPluss/LexxHard-MainBoard-Updator.git).
+Use [LexxPluss/LexxHard-SensorControlBoard-Updator](https://github.com/LexxPluss/LexxHard-SensorControlBoard-Updator.git).
 
 The firmware is automatically updated when the robot is turned on again after executing the following command.
 ```bash
-$ rosrun mainboard_updator mainboard_updator LexxHard-MainBoard-Firmware-Update-v?.?.?.bin
+$ rosrun SensorControlBoard_updator SensorControlBoard_updator LexxHard-SensorControlBoard-Firmware-Update-v?.?.?.bin
```

Comment:
I think it would be better to make it a comment.
SensorControlBoard_updator is missing.
( LexxHard-MainBoard-Updator is [here](
https://github.com/LexxPluss/LexxHard-MainBoard-Updator). )

LexxHard-MainBoard-Updator should have sent the updater via rostopic. You may feel uncomfortable, but continuing to use LexxHard-MainBoard-Updator may be an option.

Reply:
I removed the description.

@shujima Do we have a ticket to develop an update tool for the SensorControlBoard?